cmd/vendor_pull.go

```
@@ -27,12 +27,12 @@ var vendorPullCmd = &cobra.Command{
 }
 
 func init() {
-	vendorPullCmd.PersistentFlags().StringP("component", "c", "", "Only vendor the specified component: atmos vendor pull --component <component>")
+	vendorPullCmd.PersistentFlags().StringP("component", "c", "", "Only vendor the specified component: atmos vendor pull --component &ltcomponent&gt")
 	vendorPullCmd.RegisterFlagCompletionFunc("component", ComponentsArgCompletion)
```

golangci-lint (1.62.2)

Error return value of `vendorPullCmd.RegisterFlagCompletionFunc` is not checked

(errcheck)